samples/MemoryCacheConcurencySample/project.json

```diff
@@ -1,20 +1,22 @@
 {
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+
   "dependencies": {
     "Microsoft.Extensions.Caching.Memory": "1.0.0-*"
   },
-  "commands": {
-    "MemoryCacheConcurencySample": "MemoryCacheConcurencySample"
```

Comment:
Do we have tooling yet that can launch the samples without a command? If not, leave the commands for now.